### PR TITLE
RFC for PipeCD ECS Application

### PR DESCRIPTION
**Pull request by khanhtc1202**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by nakabonne** [comment]
Looks cool
/lgtm

File: docs/rfcs/0004-ecs-deployment.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+      # If this stage finds any not good metrics of the new version,
+      # a rollback process to the previous version will be executed.
+      - name: ANALYSIS
+      # Change the traffic routing state where
+      # the new version will receive 100% of the traffic.
+      - name: ECS_TRAFFIC_ROUTING
+        with:
+          canary: 100
+```
+
```
Comment (nghialv) [inline docs/rfcs/0004-ecs-deployment.md:79]: Please add more examples for quick sync, canary, blue-green to verify the definition of stages.

**Comment by khanhtc1202** [comment]
@nghialv @nakabonne thanks for your comment 🙏 I added some pipe yaml config examples for canary and blue/green release deployment, PTAL 👀

**Comment by nghialv** [comment]
👀

**Pull request merged**